src/HealthChecks.AzureServiceBus/AzureServiceBusHealthCheck.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Concurrent;
+using Azure.Core;
+using Azure.Messaging.ServiceBus.Administration;
+
+namespace HealthChecks.AzureServiceBus
+{
+    public abstract class AzureServiceBusHealthCheck
+    {
+        protected static readonly ConcurrentDictionary<string, ServiceBusAdministrationClient>
+            ManagementClientConnections = new();
+
+        private string ConnectionString { get; }
+
+        protected string Prefix => ConnectionString ?? Endpoint;
+
+        private string Endpoint { get; }
+        private TokenCredential TokenCredential { get; }
+
```

Comment:
nit: extra blank line here, but missing blank line between Endpoint and TokenCredential

Reply:
fixed